app/src/main/java/org/wikipedia/history/HistoryEntry.kt

```diff
@@ -20,12 +20,12 @@ import java.util.*
 @TypeParceler<Date, DateParceler>()
 @Entity
 class HistoryEntry(
```

Comment:
Would you mind adding a comment here saying something like "TODO: change these properties back to `val` when HistoryEntry is no longer serializable. (i.e. when we update Tabs to be in the database instead of Prefs)"

Reply:
Of course!